docs/content/users/configuration/config_yaml.md

```diff
@@ -1,76 +1,561 @@
+
+Whether to skip mounting project into web container.
+
+| Type | Default | Usage
+| -- | -- | --
+| :octicons-file-directory-16: project | `false` | Can be `true` or `false`.
+
+!!!warning "Advanced users only!"
+    When `true`, project will not be mounted by DDEV into the web container. Enables experimentation with alternate file mounting strategies.
+
+## `nodejs_version`
+
+Node.js version for the web container’s “system” version.
+
+| Type | Default | Usage
+| -- | -- | --
+| :octicons-file-directory-16: project | current LTS version | Can be `14`, `16`, or `18`.
+
+`nvm` is also available inside the container and via `ddev nvm`, and can be set to any valid version including much older ones.
+
+## `omit_containers`
+
+Containers that should not be loaded automatically for one or more projects.
+
+| Type | Default | Usage
+| -- | -- | --
+| :octicons-file-directory-16: project<br>:octicons-globe-16: global | `[]` | **For projects**, can include `db`, `dba`, and `ddev-ssh-agent`.<br>**Globally**, can include `dba` and `ddev-ssh-agent` only.
+
+Example: `omit_containers: [db, dba, ddev-ssh-agent]` starts the project without a `db` container, phpMyAdmin, and SSH agent. Some containers can be omitted globally in `~/.ddev/global_config.yaml` and the result is additive; all containers named in both places will be omitted.
+
+!!!warning
+    Omitting the `db` container will cause database-dependent DDEV features to be unstable.
+
+## `override_config`
+
+Whether to override config values instead of merging.
+
+| Type | Default | Usage
+| -- | -- | --
+| :octicons-file-directory-16: project | `false` | Can be `true` or `false`.
+
+When `true`, the `config.*.yaml` file with the option will have its settings *override* rather than *merge with* others. Allows statements like `nfs_mount_enabled: false` or `additional_hostnames: []` to work.
+
+## `php_version`
+
+The PHP version the project should use.
+
+| Type | Default | Usage
+| -- | -- | --
+| :octicons-file-directory-16: project | `8.0` | Can be `5.6`, `7.0`, `7.1`, `7.2`, `7.3`, `7.4`, `8.0`, `8.1`, or `8.2`.
+
+You can only specify the major version (`7.3`), not a minor version (`7.3.2`), from those explicitly available.
+
+## `phpmyadmin_https_port`
+
+Port for project’s phpMyAdmin HTTPS URL.
+
+| Type | Default | Usage
+| -- | -- | --
+| :octicons-file-directory-16: project | `8037` | Can be changed to avoid a port conflict.
+
+## `phpmyadmin_port`
+
+Port for project’s phpMyAdmin HTTP URL.
+
+| Type | Default | Usage
+| -- | -- | --
+| :octicons-file-directory-16: project | `8036` | Can be changed to avoid a port conflict.
+
+## `project_tld`
+
+Default TLD to be used for a project’s domains, or globally for all project domains.
+
+| Type | Default | Usage
+| -- | -- | --
+| :octicons-file-directory-16: project<br>:octicons-globe-16: global | `ddev.site` | Can be changed to any TLD you’d prefer.
+
+## `required_docker_compose_version`
+
+Specific docker-compose version for download.
```

Comment:
This will only be used in specific troubleshooting cases. This option should not be used.

Reply:
[Fixed](https://github.com/drud/ddev/pull/4332/commits/5258a6541398d0521dde8f64b5b151bacbf53191).